### PR TITLE
Static analysis

### PR DESCRIPTION
**Pull request by netromdk**
I did some static analysis using Clang's scan-build and CppCheck.

### CODE REVIEW

**Comment by netromdk** [comment]
For some reason the OSX Travis build fails to download its Qt dependency.. And then takes more than 10 minutes without output so the build fails. I will try to restart it again again.

**Comment by netromdk** [comment]
Came through finally. :)

File: src/AnsiEscapeCodeHandler.h [hunk 1]
```diff
@@ -30,11 +30,21 @@ namespace Utils {
 class FormattedText {
 public:
     FormattedText() { }
-    FormattedText(const FormattedText &other) : text(other.text), format(other.format) { }
+
+    FormattedText(const FormattedText &other) {
```
Comment (raoulh) [inline src/AnsiEscapeCodeHandler.h]: } on newline please, to follow the project code style

File: src/AnsiEscapeCodeHandler.h [hunk 2]
```diff
@@ -30,11 +30,21 @@ namespace Utils {
...
+        *this = other;
+    }
+
     FormattedText(const QString &txt, const QTextCharFormat &fmt = QTextCharFormat()) :
         text(txt), format(fmt)
     { }
 
+    FormattedText &operator=(const FormattedText &other) {
```
Comment (raoulh) [inline src/AnsiEscapeCodeHandler.h]: same

File: src/AppDaemon.cpp [hunk 3]
```diff
@@ -36,6 +38,7 @@ AppDaemon::AppDaemon(int &argc, char **argv):
 
 bool AppDaemon::initialize()
 {
+    assert(!localLogServer);
```
Comment (raoulh) [inline src/AppDaemon.cpp]: use Q_ASSERT instead of assert

File: src/zxcvbn-c/zxcvbn.c [hunk 4]
```diff
@@ -638,9 +638,9 @@ static void DoDictMatch(const uint8_t *Passwd, int Start, int MaxLen, DictWork_t
     for(Len = 0; *Passwd && (Len < MaxLen); ++Len, ++Passwd)
     {
         uint8_t c;
-        int w, x, y, z;
```
Comment (raoulh) [inline src/zxcvbn-c/zxcvbn.c]: Actually this is not our code, I wouldn't touch it

File: src/AnsiEscapeCodeHandler.h (hunk 1 above)
Comment (netromdk) [reply src/AnsiEscapeCodeHandler.h]: Sorry about that.

File: src/AppDaemon.cpp (hunk 3 above)
Comment (netromdk) [reply src/AppDaemon.cpp]: Will do.

**Comment by raoulh** [comment]
Thanks you for your work!

**Pull request merged**